vocode/streaming/agent/state_agent.py

```diff
@@ -453,6 +442,8 @@ def __init__(
         self,
         agent_config: CommandAgentConfig,
         logger: Optional[logging.Logger] = None,
+        *,
+        events_manager: Optional[EventsManager] = None,
```

Comment:
i think we can remove this now

Reply:
Yep tested it now and the twilio call has no new error I think, here are some of the unrelated errors, and I got the json transcript email postcall workflow. Also tested ws ui again ![image](https://github.com/user-attachments/assets/a299cb17-33c4-405f-a45b-3ec8a5f578d6) .
```
...
DEBUG:telephony_app.main:[EAschoNVjMLnkun9NrLEag] Broadcasting interrupt                                                                                                                 
DEBUG:telephony_app.main:Sending clear message: {'event': 'clear', 'streamSid': 'MZ7b32078540f8df71743ceb86cecfb5c3'}                                                                    
DEBUG:telephony_app.main:Sending clear message: {'event': 'clear', 'streamSid': 'MZ7b32078540f8df71743ceb86cecfb5c3'}                                                                    
DEBUG:telephony_app.main:[EAschoNVjMLnkun9NrLEag] Finished broadcasting interrupt, num_interrupts: 0
INFO:telephony_app.main:[EAschoNVjMLnkun9NrLEag] Buffer changed on interrupt, putting stashed buffer back                                                                                
ERROR:telephony_app.main:[EAschoNVjMLnkun9NrLEag] Error making request: [Errno -2] Name or service not known
INFO:telephony_app.main:[EAschoNVjMLnkun9NrLEag] Transcription event put in output queue                                                                                                 
DEBUG:telephony_app.main:Responding to transcription                                                                                                                                     
INFO:telephony_app.main:current intent description: continue                                                                                                                             
...
ERROR:telephony_app.main:[EAschoNVjMLnkun9NrLEag] Error making request: [Errno -2] Name or service not known                                                                             
...
DEBUG:telephony_app.main:Action description: send a booking link via sms
ERROR:telephony_app.main:Action config not found. Simulating action completion. Error: 'NoneType' object has no attribute 'starting_phrase'
ERROR:root:Error in compose_action 1 validation error for StateAgentTranscriptActionFinish
runtime_inputs
...
INFO:telephony_app.main:[CallType.INBOUND:84849] Lead:[inaudible]
ERROR:telephony_app.main:No intent task
INFO:telephony_app.main:Resuming from previous resume
ERROR:vocode.streaming.utils.worker:InterruptibleWorker
Traceback (most recent call last):
File "/code/local_vocode/vocode-python-main/vocode/streaming/utils/worker.py", line 228, in _run_loop
await self.current_task
File "/code/local_vocode/vocode-python-main/vocode/streaming/agent/base_agent.py", line 357, in process
should_stop = await self.handle_generate_response(
File "/code/local_vocode/vocode-python-main/vocode/streaming/agent/base_agent.py", line 253, in handle_generate_response
function_call = await self.respond_with_functions(
File "/code/local_vocode/vocode-python-main/vocode/streaming/agent/base_agent.py", line 518, in respond_with_functions
response = await self.generate_completion(
File "/code/local_vocode/vocode-python-main/vocode/streaming/agent/state_agent.py", line 764, in generate_completion
self.resume_task = asyncio.create_task(self.resume(human_input))
TypeError: 'NoneType' object is not callable
...
```